interface/themes/oe-styles/style_light.scss

```diff
@@ -1,9 +1,12 @@
 /* Bootstrap Code */
-// @import "../default-variables";
+
 $black: #222;
 $close-color: $black;
 $close-text-shadow: 0 1 0 $black;
 $gray: #cdcdcd;
+
+@import "../default-variables";
```

Comment:
This worries me a bit! It was patched out for a reason which I don't know. Pathing probably.
Light theme is kind of a special theme that broke with our older color themes so it renders differently in many ways.

I'd say you're correct but I still worry. Need to test well.